src/constants/bridge.ts

```diff
@@ -0,0 +1,30 @@
+import { ChainId } from '@fuseio/fuse-swap-sdk'
+
+export const DEFAULT_CONFIRMATIONS_LIMIT = 2
+
+export const BASIC_BRIDGE_TOKENS = {
```

Comment:
@mul53 not sure it's a good name. better rename to something like custom_bridge_tokens